micrometer/tasks/task-demo-metrics-prometheus/README.md

```diff
@@ -0,0 +1,141 @@
+
+# How to monitor Spring Cloud Tasks
```

Comment:
If you agree with the suggestions on Influx README, please also apply the same changes in this README as well. Almost same except of course the influx vs. prometheus callouts.